device_info_plus plugin is producing build warnings on macos

<!-- Thank you for using this plugin!

     If you are looking for support, please check out our documentation
     or consider asking a question on Stack Overflow:
      * https://flutter.dev/
      * https://api.flutter.dev/
      * https://stackoverflow.com/questions/tagged/flutter?sort=frequent

     If you have found a bug or if our documentation doesn't have an answer
     to what you're looking for, then fill our the template below.
-->

## System info

Issue occurs on: macos
Plugin name: device_info_plus
Plugin version: 3.1.0
Flutter doctor output:

```
see below
```

## Steps to Reproduce

<!--
     Please tell us exactly how to reproduce the problem you are running into.

     Please attach a small application (ideally just one main.dart file) that
     reproduces the problem. You could use https://gist.github.com/ for this.

     If the problem is with your application's rendering, then please attach
     a screenshot and explain what the problem is.
-->

When `device_info_plus` plugin is used in the application, build on macos prodices the following warnings when compiling Swift code:

```
/Users/eu/.pub-cache/hosted/pub.dartlang.org/device_info_plus_macos-2.2.0/macos/Classes/DeviceInfoPlusMacosPlugin.swift:42:27: warning: expression implicitly coerced from 'String?' to 'Any'
            "systemGUID": guid
                          ^~~~
/Users/eu/.pub-cache/hosted/pub.dartlang.org/device_info_plus_macos-2.2.0/macos/Classes/DeviceInfoPlusMacosPlugin.swift:42:27: note: provide a default value to avoid this warning
            "systemGUID": guid
                          ^~~~
                               ?? <#default value#>
```

## Logs

<!--
      Run your application with `flutter run --verbose` and attach all the
      log output below between the lines with the backticks. If there is an
      exception, please see if the error message includes enough information
      to explain how to solve the issue.
-->

```
nothing interesting there
```

<!--
     Run `flutter analyze` and attach any output of that command below.
     If there are any analysis errors, try resolving them before filing this issue.
-->

```
No issues found! (ran in 1.4s)
```

<!-- Finally, paste the output of running `flutter doctor -v` here. -->

```
[✓] Flutter (Channel stable, 2.5.3, on Mac OS X 10.15.7 19H1419 darwin-x64, locale en-CA)
    • Flutter version 2.5.3 at /Users/eu/Downloads/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (5 days ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at .../android-sdks
    • Platform android-31, build-tools 30.0.3
    • ANDROID_HOME = .../android-sdks
    • ANDROID_SDK_ROOT = .../android-sdks
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode 12.4.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] IntelliJ IDEA Community Edition (version 2021.2.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • Mac OS X 10.15.7 19H1419 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 94.0.4606.81

• No issues found!
```

--- a/packages/device_info_plus/device_info_plus_macos/macos/Classes/DeviceInfoPlusMacosPlugin.swift
+++ b/packages/device_info_plus/device_info_plus_macos/macos/Classes/DeviceInfoPlusMacosPlugin.swift
@@ -7,7 +7,7 @@ public class DeviceInfoPlusMacosPlugin: NSObject, FlutterPlugin {
         let instance = DeviceInfoPlusMacosPlugin()
         registrar.addMethodCallDelegate(instance, channel: channel)
     }
-    
+
     public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
         switch call.method {
         case "getMacosDeviceInfo":
@@ -16,7 +16,7 @@ public class DeviceInfoPlusMacosPlugin: NSObject, FlutterPlugin {
             result(FlutterMethodNotImplemented)
         }
     }
-    
+
     private func handleDeviceInfo(result: @escaping FlutterResult)-> Void{
         let computerName = Sysctl.hostName
         let hostName = Sysctl.osType
@@ -28,7 +28,7 @@ public class DeviceInfoPlusMacosPlugin: NSObject, FlutterPlugin {
         let memorySize = Sysctl.memSize
         let cpuFrequency = Sysctl.cpuFreq
         let guid = SystemUUID.getSystemUUID()
-        
+
         result([
             "computerName": computerName,
             "hostName": hostName,
@@ -40,6 +40,6 @@ public class DeviceInfoPlusMacosPlugin: NSObject, FlutterPlugin {
             "memorySize": memorySize,
             "cpuFrequency": cpuFrequency,
             "systemGUID": guid
-        ])
+        ] as [String: Any?])
     }
 }

